eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -228,11 +228,34 @@ jobs:
         condition: and(succeeded(), eq(variables['ShouldRunSourceTests'],'true'))
         displayName: 'Update versions for source build'
 
+      - pwsh: |
```

Comment:
In CI, we build all modules originally.  How do you use TestFromSource to indicate whether it is from maven or from local module?

Reply:
There is a set-version in python that updates all the pom.xmls to use current-version.